### PR TITLE
Rework some states in English translation [need feedback]

### PR DESCRIPTION
**Pull request by c727**
As we discussed on Discord, maybe we can rework some of our reference-texts to make them more context specific.
Also some translators started to translate in context.

Other suggestions are very welcome

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -41,28 +41,28 @@
...
       "motion": {
-        "off": "[%key:state::binary_sensor::gas::off%]",
-        "on": "[%key:state::binary_sensor::gas::on%]"
+        "off": "Nothing",
+        "on": "Motion detected"
       },
       "occupancy": {
-        "off": "[%key:state::binary_sensor::gas::off%]",
-        "on": "[%key:state::binary_sensor::gas::on%]"
+        "off": "Free",
```
Comment (armills) [inline src/translations/en.json]: I'd prefer Empty/Occupied here.

File: src/translations/en.json [hunk 2]
```diff
@@ -41,28 +41,28 @@
         "on": "Wet"
       },
       "gas": {
-        "off": "Clear",
-        "on": "Detected"
+        "off": "Nothing",
```
Comment (armills) [inline src/translations/en.json]: I don't like changing clear to nothing myself, but I'll let others weigh in. I also think just changing Detected to Gas detected is unnecessary extra verbosity. I'd only want to change where we can use more specific words as was done for occupancy.

**Comment by armills** [comment]
I'd also just add that context specific translations even on keys where we don't do it in English is perfectly acceptable, and the reason we've laid out the translations with this schema.

File: src/translations/en.json (hunk 1 above)
Comment (balloob) [reply src/translations/en.json]: The problem is that we need to have it fix in a badge, which means like 5/6 characters.

File: src/translations/en.json (hunk 2 above)
Comment (balloob) [reply src/translations/en.json]: Clear will not get cut off in the label badge.

File: src/translations/en.json (hunk 1 above)
Comment (armills) [reply src/translations/en.json]: Binay sensor state text is actually only shown in the more-info, at least currently.

Comment (balloob) [reply src/translations/en.json]: Ah, good point.

File: src/translations/en.json [hunk 3]
```diff
@@ -41,28 +41,28 @@
...
+        "on": "Gas detected"
       },
       "motion": {
-        "off": "[%key:state::binary_sensor::gas::off%]",
-        "on": "[%key:state::binary_sensor::gas::on%]"
+        "off": "Nothing",
```
Comment (andrey-git) [inline src/translations/en.json]: I think "Nothing" is unclear. I prefer the "Off" text :)

File: src/translations/en.json (hunk 1 above)
Comment (andrey-git) [reply src/translations/en.json]: Empty sounds good

**Comment by c727** [comment]
I also like "Off", it's easier to translate .Well let's see what the users say about the translated texts after next release. There will be many out of context translations

**Pull request closed without merging**